draft-ietf-quic-transport.md

```diff
@@ -2733,8 +2733,8 @@ a small packet might result in Stateless Reset not being useful in detecting
 cases of broken connections where only very small packets are sent; such
 failures might only be detected by other means, such as timers.
 
-An endpoint that wants to improve the chances that it triggers a Stateless Reset
-when its peer loses state can pad all packets it sends to 38 octets.
+An endpoint can increase the odds that a packet will trigger a Stateless Reset
```

Comment:
Does increasing it above 38 not ensure it will trigger a Stateless Reset?  If so, I'd prefer "An endpoint can ensure that a packet will trigger..."

Reply:
I would hope that stateless reset is an optional feature and when it is on, resets can be send at senders discretion, e.g. implementing rate limiting.

Reply:
@mikkelfj is right, this doesn't guarantee that you get a stateless reset.  The point here is that the odds of receiving one for packets below 38 octets is much lower.